### DISCUSSION TITLE
How to make a wonderful profile on Github

### DISCUSSION THREAD

**Discussion by km1220** [community/community#36432] [Other Feature Feedback, Questions, & Ideas]
Hi everyone!
I'm one of the developer in the world 
and also love Github very much as well.
Many of clients and other developers value us via the profile so making a wonderful and perfect profile is very important.
Of course, there are many discussions already on this Github discussion community.
If you want to achieve some badges you can search about it.
If you want a amazing Readme file you can also do that by searching.

Anyway what I want to say to you is trying to make your profile looks better is a big step for your success.
So please don't depreciate it.
That's all. Thank you very much for reading my text.

**Comment by ossanna16** [comment]
Hey there @km1220 and thank you for the advice! I'm glad you found our community 🙂

**Comment by km1220** [comment]
thank very much @ossanna16 
let's keep in touch
if you want to ask something, feel free to contact with me :)
cheers
